apps/design-system/src/subjects/views/execution-details/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { ExecutionHeader, ExecutionInfo, ExecutionTabs, ExecutionTree } from '@harnessio/ui/views'
+
+import { elements, logs, stages } from './mocks/mock-data'
+
+export const ExecutionDetailsView = () => {
+  return (
+    <div className="flex flex-col">
+      <ExecutionTabs />
+      <ExecutionHeader commitName="8fbru3ix" branchName="master" title="#311. Alerting docs: adds sns integration" />
+      <div className="grid p-4" style={{ gridTemplateColumns: '1fr 3fr' }}>
+        <ExecutionTree
+          defaultSelectedId="initialize"
+          elements={elements}
+          onSelectNode={({ parentId, childId }: { parentId: string; childId: string }) => {
```

Comment:
we should support n-levels of hierarchy here by sending `a.b.c.d...` fully qualiified path to a node instead.

Reply:
Yes, taking it up as a follow up item.